metecho/api/models.py

```diff
@@ -378,9 +365,6 @@ def save(self, *args, **kwargs):
             )
             self.latest_sha = repo.branch(self.branch_name).latest_sha()
 
-        if not self.github_users:
-            self.queue_populate_github_users(originating_user_id=None)
-
```

Comment:
@jgerigmeyer I removed this call to the job because we `save()` when finalizing the job so this just re-queued the job again. I assume this was here so GitHub users are fetched in the background after a new project is added in the admin, but I think we can have the frontend queue the job as we do in other places when the required data is empty (in this case `project.github_users`). What do you think?